frigate/comms/mqtt.py

```diff
@@ -168,9 +168,11 @@ def _on_disconnect(
 
     def _start(self) -> None:
         """Start mqtt client."""
+        logger.info("MQTT transport mechanism: %s" % str(self.mqtt_config.transport))
```

Comment:
I don't think this should be an info log.

Reply:
ok, debug maybe or get rid of this line ?

Reply:
Debug seems fine